### ISSUE TITLE
Does it work's actually?

### ISSUE THREAD

**Issue by jmmccota** [NikolayIT/RatioMaster.NET#37]
Have any plan to update? Add new features or new clients?

Thanks!

**Comment by NikolayIT** [comment]
Yes: RatioMaster.NET 1.0 was released on 2026-09-06 (https://github.com/NikolayIT/RatioMaster.NET/releases/tag/v1.0.0). It is a complete rewrite for Windows, Linux and macOS with https support and new client emulations (qBittorrent 5.2.3 is the default; uTorrent 3.6.0 and 3.5.5 follow in 1.1).
